### PR TITLE
fix: added the numpy.digitize function on JAX frontend  

### LINKED ISSUE #23105: digitize

### PR DESCRIPTION
**Pull request by HarbdhulQuadri**
# PR Description
fix: added the numpy.digitize function on JAX frontend

## Related Issue

Close #23105 

## Checklist

- [x] Did you add a function?
- [x] Did you add the tests?
- [x] Did you follow the steps we provided?

### CODE REVIEW

**Review by saeedashrraf — CHANGES_REQUESTED** [review]

File: ivy/functional/frontends/jax/numpy/statistical.py [hunk 1]
```diff
@@ -294,7 +294,19 @@ def nanmean(a, axis=None, dtype=None, out=None, keepdims=False, *, where=None):
         ivy.full_like(not_nan_mask_count1, ivy.nan),
     )
     return ivy.divide(array_sum1, count_zero_handel)
-
+    
+@handle_jax_dtype
```
[the remark this answers is not in the document]
Comment (HarbdhulQuadri) [reply ivy/functional/frontends/jax/numpy/statistical.py]: I've removed it

**Comment by HarbdhulQuadri** [comment]
can you please check now ? thanks

**Review by saeedashrraf — CHANGES_REQUESTED** [review]
Hello @HarbdhulQuadri , it seems like the test implemented does not correctly follow the way provided in the guide. Can you please fix that and update it to follow the latest guide [here](https://unify.ai/docs/ivy/overview/deep_dive/ivy_frontends_tests.html#ivy-frontend-tests)? Thank you for your contribution!

**Comment by HarbdhulQuadri** [comment]
Ok thanks for following up, is it the test alone that has issues or there’s a list and how can I reach out to you faster so I can make great progress on contributing

**Comment by saeedashrraf** [comment]
Hello @HarbdhulQuadri, I apologize for the delay as it seems like your PR somehow slipped under the radar. Once you are done with the requested changes, just request a review from me and I will let you know if we can merge the PR. Thanks!

**Comment by HarbdhulQuadri** [comment]
but is it compulsory i write test for the function i implemented...

**Comment by saeedashrraf** [comment]
As per the [contributors guide](https://unify.ai/docs/ivy/overview/contributing/open_tasks.html#frontend-apis), you should write the tests for the function you implemented and make sure that the function implementation passes the tests successfully. Thanks in advance!

**Comment by ivy-seed** [comment]
This PR has been labelled as stale because it has been inactive for more than 7 days. If you would like to continue working on this PR, then please add another comment or this PR will be closed in 7 days.

**Comment by ivy-seed** [comment]
This PR has been closed because it has been marked as stale for more than 7 days with no activity.

**Pull request closed without merging**